lms/djangoapps/courseware/views/views.py

```diff
@@ -110,7 +110,8 @@
 from openedx.features.course_experience import (
     COURSE_ENABLE_UNENROLLED_ACCESS_FLAG,
     UNIFIED_COURSE_TAB_FLAG,
-    course_home_url_name
+    course_home_url_name,
+    RELATIVE_DATES_FLAG,
```

Comment:
I _thought_ you can't end an import with a comma. Could be wrong, but just something to watch out for.